connectors/cassandra/translator-cassandra/src/main/java/org/teiid/translator/cassandra/CassandraMetadataProcessor.java

```diff
@@ -61,7 +69,14 @@ public void process(MetadataFactory factory, CassandraConnection connection) thr
 	private void addTable(MetadataFactory factory, TableMetadata columnFamily) {
 		Table table = factory.addTable(columnFamily.getName());
 		addColumnsToTable(factory, table, columnFamily);
-		addPrimaryKey(factory, table, columnFamily);
+        addPrimaryKey(factory, table, columnFamily);

```

Comment:
This change is from earlier commit, you need to rebase your commit from master, such that it only pulls in your changes